File: Problem_15/main.py
def factorial(number):

	if number == 0:
		return 1
	return number*factorial(number-1)

def main():

	system_size = 20
	paths = int(factorial(2*system_size)/(factorial(system_size)**2))
	print (paths)

if __name__ == '__main__':
	main()

File: Problem_15/main.js
function factorial(number) {
    if (number === 0)
        return 1;
    return number * factorial(number - 1);
}

let L = 20;
let A = factorial(L + L);
let B = factorial(L);

console.log(A / (B * B));

// time 0.310 s

